### PR TITLE
DEV: migrates themes-list-item to gjs

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: app/assets/javascripts/admin/addon/components/themes-list-item.gjs [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  }
+
+  @action
+  toggleChildrenExpanded(event) {
+    event?.preventDefault();
+    this.childrenExpanded = !this.childrenExpanded;
+  }
+
+  <template>
+    {{! template-lint-disable no-nested-interactive }}
```
Comment (jjaffeux) [inline app/assets/javascripts/admin/addon/components/themes-list-item.gjs:72]: I can't avoid this without refactoring a lot more and changing the UI/UX

**Review by CvX — APPROVED** [review]

**Comment by CvX** [comment]
(the spec failure looks related)

**Comment by jjaffeux** [comment]
> (the spec failure looks related)

yep on it

**Comment by jjaffeux** [comment]
Hmm actually green 🤔

File: app/assets/javascripts/admin/addon/components/themes-list-item.gjs (hunk 1 above)
Comment (Amin8087) [reply app/assets/javascripts/admin/addon/components/themes-list-item.gjs:76]: > I can't avoid this without refactoring a lot more and changing the UI/UX

**Pull request merged**